modules/oadp-kopia-algorithms-benchmarking.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/overriding-kopia-algorithms.adoc          
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-kopia-algorithms-benchmarking_{context}"]
+= Benchmarking Kopia hashing, encryption, and splitter algorithms
+
+You can run Kopia commands to benchmark the hashing, encryption, and splitter algorithms. Based on the benchmarking results, it is possible to select the most suitable algorithm for your workload. In this procedure, you run the Kopia benchmarking commands from a pod on the cluster. The benchmarking results can vary depending on CPU speed, available RAM, disk speed, current I/O load, and so on.
```

Comment:
```suggestion
...it is possible to select the most suitable algorithm for your workload.
```

Does "it" refer to Kopia commands or the algorithms?

Reply:
Have reworded it to 'you can select..'.